app/views/foreman_ansible/hosts/_tab_content.html.erb

```diff
@@ -1,4 +1,4 @@
 <div class='tab-pane' id='ansible_roles'>
-  <%= multiple_checkboxes f, :ansible_roles, @host, ansible_roles,
+  <%= multiple_checkboxes f, :ansible_roles, @host, AnsibleRole.all,
```

Comment:
If you want to do this, you'll have to add permissions to AnsibleRole then call .authorized here